src/lib/y2partitioner/actions/go_to_device_tab.rb

```diff
@@ -46,7 +46,7 @@ def run
 
         # First, pretend the user visited the device and then the tab...
         state = UIState.instance
-        state.go_to_tree_node(target_page)
+        state.select_page([target_page.id])
```

Comment:
Isn't it possible to use here `#tree_path`? `        state.select_page(target_page.tree_path)`

Reply:
Sure!